solutions/level1/library.rb

```diff
@@ -0,0 +1,14 @@
+class Library
+  attr_accessor :books
+
+  def initialize
+    @books = []
+  end
+
+  def create_books(books_hash)
+    books_hash.transform_keys(&:to_sym)[:books].each do |book_hash|
```

Comment:
instead of doing twice the keys transformation, it might be better to see how you can use `symbolize_names` when parsing the json